<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1420</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>1119</x>
      <y>11</y>
      <width>291</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="nodeInfoPanel">
       <property name="title">
        <string>Свойства выбранного узла</string>
       </property>
       <widget class="QWidget" name="formLayoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>271</width>
          <height>224</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="nodeIdEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="nodeNameEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="nodeXEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="nodeYEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Стоимость:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="nodeCostEdit"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="edgeInfoPanel">
       <property name="title">
        <string>Свойства выбранного ребра</string>
       </property>
       <widget class="QWidget" name="formLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>271</width>
          <height>262</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Имя</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edgeNameEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Проп. спос</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Длина</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edgeLengthEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Стоимость:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edgeCostEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Поток (Мбит/с)</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edgeDelayEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="edgeCapacityComboBox"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Загрузка (%):</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="edgeUtilizationEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Задержка (мс):</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="edgeCalculatedDelayEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1081</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="DrawingCanvas" name="drawingCanvas" native="true"/>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="debugOutputTextEdit">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>592</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addNodeButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>740</y>
      <width>289</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить узел</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="moveModeCheckBox">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>712</y>
      <width>289</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Режим передвижения</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1420</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionLoadFromExcel"/>
    <addaction name="actionLoadFromJson"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Сохранить как</string>
    </property>
    <addaction name="actionSaveAsJson"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Этапы</string>
    </property>
    <addaction name="actionCalculateRoutes"/>
    <addaction name="actionCalculateFlows"/>
    <addaction name="actionEvaluateProject"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadFromExcel">
   <property name="text">
    <string>Загрузить из Excel</string>
   </property>
  </action>
  <action name="actionLoadFromJson">
   <property name="text">
    <string>Загрузить из JSON</string>
   </property>
  </action>
  <action name="actionSaveAsJson">
   <property name="text">
    <string>Сохранить в JSON</string>
   </property>
  </action>
  <action name="actionCalculateRoutes">
   <property name="text">
    <string>Выбрать маршруты</string>
   </property>
  </action>
  <action name="actionCalculateFlows">
   <property name="text">
    <string>Загрузить нагрузку</string>
   </property>
  </action>
  <action name="actionEvaluateProject">
   <property name="text">
    <string>Оценить проект</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawingCanvas</class>
   <extends>QWidget</extends>
   <header>drawing_canvas</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
